packages/suite-desktop-core/webpack/core.webpack.config.ts

```diff
@@ -0,0 +1,159 @@
+        'bufferutil',
+        'memcpy',
+        'utf-8-validate',
+        'osx-temperature-sensor',
+    ],
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /node_modules/,
+                use: ['babel-loader'],
+            },
+            {
+                test: /\.ts$/,
+                exclude: /node_modules/,
+                use: {
+                    loader: 'babel-loader',
+                    options: {
+                        presets: ['@babel/preset-typescript'],
+                    },
+                },
+            },
+        ],
+    },
+    resolve: {
+        modules: ['node_modules'],
+        mainFields: ['module', 'main'],
+        extensions: ['.ts', '.js'],
+        alias: {
+            '@emurgo/cardano-serialization-lib-nodejs': '@emurgo/cardano-serialization-lib-browser',
+            '@trezor/connect$': '@trezor/connect/src/index', // alternative for "module": "src/index" in connect's package.json
+        },
+    },
+    performance: {
+        hints: false,
+    },
+    optimization: {
+        splitChunks: {
+            chunks: 'all',
+            name(_: any, chunks: any) {
+                return chunks.length === 1
+                    ? chunks[0].name
+                    : `shared/${chunks.map((item: any) => item.name.split('/').pop()).join('~')}`;
+            },
+        },
+        minimizer: [
+            new TerserPlugin({
+                extractComments: false,
+                terserOptions: {
+                    format: {
+                        comments: false,
+                    },
+                },
+            }),
+        ],
+    },
+    plugins: [
+        new webpack.DefinePlugin({
+            'process.env.NODE_ENV': JSON.stringify(NODE_ENV),
+            'process.env.COMMITHASH': JSON.stringify(gitRevision),
+            'process.env.APP_PUBKEY': JSON.stringify(appKey),
+            'process.env.PROTOCOLS': JSON.stringify(uriSchemes),
+            'process.env.VERSION': JSON.stringify(suiteVersion),
+            'process.env.SENTRY_RELEASE': JSON.stringify(sentryRelease),
+            'process.env.SUITE_TYPE': JSON.stringify(PROJECT),
+            'process.env.IS_CODESIGN_BUILD': `"${isCodesignBuild}"`, // to keep it as string "true"/"false" and not boolean
+            'process.env.NODE_BACKEND': JSON.stringify('js'), // TODO is this equivalent to build-inject?
+        }),
+    ],
+
+    // We are using WASM package - it's much faster (https://github.com/Emurgo/cardano-serialization-lib)
+    // This option makes it possible
+    // Unfortunately Cardano Serialization Lib triggers webpack warning:
+    // "Critical dependency: the request of a dependency is an expression" due to require in generated wasm module
+    // https://github.com/Emurgo/cardano-serialization-lib/issues/119
+    experiments: { asyncWebAssembly: true },
+    ignoreWarnings: [{ module: /cardano-serialization-lib-browser/ }],
+};
```

Comment:
just thinking, whether it would make sense to import from 
https://github.com/trezor/trezor-suite/blob/develop/packages/connect-iframe/webpack/prod.webpack.config.ts#L192 
and spread it here. 

of course it doesn't make sense at this very moment, why would suite-dsktop import from connect-iframe. it doesn't have anything to do with iframe. but I am now considering moving connect webpacks around in #9896

Reply:
It could make sense to have some base connect webpack and reuse it throughout the repo, but a lot of things must be different here and there, so I'd definitely wait after merging my and your PRs.